go.mod

```diff
@@ -13,9 +14,11 @@ require (
 	github.com/microsoft/go-mssqldb v0.20.0
 	github.com/peterh/liner v1.2.2
 	github.com/pkg/errors v0.9.1
+	github.com/sirupsen/logrus v1.9.0
```

Comment:
>github.com/sirupsen/logrus v1.9.0 [](http://example.com/codeflow?start=1&length=33)

run build/build to update the NOTICE file

Reply:
done

Reply:
also removed the logrus dependency